tests/cargo-kani/build-rs-plus-host-with-kani-proofs/README.md

```diff
@@ -0,0 +1,10 @@
+This repo contains contains a minimal example that breaks compilation when using [kani](https://github.com/model-checking/kani), where I would expect compilation to work.
+
+Deleting the `binary/build.rs` script makes the compilation work suddenly, despite it being skipped anyways:
+
+```
+binary$ cargo kani -v
+Kani Rust Verifier 0.48.0 (cargo plugin)
+Skipped the following unsupported targets: 'build-script-build'.
```

Comment:
Admitted I also don't know what "build-script-build" means - could you enlighten me so that I can add specific text for this case (rather than throwing it into the catch-all `ignored-unsupported`?

Reply:
That target represents the `build.rs` file. https://doc.rust-lang.org/cargo/reference/build-scripts.html